_pages/2023/uss.md

```diff
@@ -6,23 +6,22 @@ permalink: /2023/uss
 
 <br>
 
-# The UNIQUE Student Symposium 2023 is coming soon!
```

Comment:
This page still has a lot of content from last year and should be updated before we merge the PR.

Reply:
Is it about the section "Important dates"?

Reply:
Not just the dates but some of the descriptions as well, for example:
1. Have we decided whether there will be posters or lighting talks or both?
2. What is the event subtitle? It is still set to last year's (Dynamics of Learning).
3. The accepted posters page should not yet be linked.
4. Is the submission deadline the same this year as well? (May 27)

Reply:
Added one more commit to fix the main page uss23:
1. changed the title
2. disabled accepted posters for now
3. fixed the important dates
4. in-person for posters and the lightnighting talks